pkg/cmd/cli/cmd/startbuild.go

```diff
@@ -242,13 +242,18 @@ func (o *StartBuildOptions) Complete(f *clientcmd.Factory, in io.Reader, out io.
 }
 
 // Run contains all the necessary functionality for the OpenShift cli start-build command
-func (o *StartBuildOptions) Run() error {
+func (o *StartBuildOptions) Run(f *clientcmd.Factory, cmd *cobra.Command) error {
 	if len(o.FromWebhook) > 0 {
 		return o.RunStartBuildWebHook()
 	}
 	if len(o.ListWebhooks) > 0 {
 		return o.RunListBuildWebHooks()
 	}
+
+	mapper, _ := f.Object(kcmdutil.GetIncludeThirdPartyAPIs(cmd))
+
+	shortOutput := kcmdutil.GetFlagString(cmd, "output") == "name"
```

Comment:
do we support any of the other output methods? (yaml/json, etc)? if not, validate this is unspecified or is `name` until we broaden to support other output printing methods